packages/fluentui/docs/src/examples/components/Carousel/BestPractices/CarouselBestPractices.tsx

```diff
@@ -8,7 +8,16 @@ const doList = [
   <Text>
     'Add textual representation for `CarouselItem`. Use `aria-label` attribute ( refer to{' '}
     {link('reported issue', 'https://bugs.chromium.org/p/chromium/issues/detail?id=1040924')} for details).
-  </Text>
+  </Text>,
+  'Provide  localized string of the "carousel" using `ariaRoleDescription` prop.',
+  'Provide label to the carousel using `ariaLabel` prop.',
+  <Box>
+    If carousel contains `navigation`:
+    <ul>
+      <li> provide label to `navigation` and to navigation item using `aria-label` attribute</li>
+      <li> add `aria-controls` attribute to navigation item referencing to `carouselItem` id </li>
```

Comment:
do we have an example that uses `aria-controls`?

Reply:
yes, already first Carousel example on the page 
https://fluentsite.z22.web.core.windows.net/components/carousel/definition
using 'aria-controls'